drivers/wifi/siwx91x/siwx91x_wifi.h

```diff
@@ -14,6 +14,8 @@
 #include "sl_si91x_types.h"
 #include "sl_si91x_protocol_types.h"
 
+#define SIWX91X_INTERFACE_MASK (0x03)
```

Comment:
For now, `SIWX91X_INTERFACE_MASK` is only used in `siwx91x_wifi.c`. I am not sure it make sense to expose it in the header file.